lib/rspec/rails/example/system_example_group.rb

```diff
@@ -78,16 +78,19 @@ def app
         def initialize(*args, &blk)
           super(*args, &blk)
           @driver = nil
+
+          self.class.before do
+            # A user may have already set the driver, so only default if driver
+            # is not set
+            driven_by(:selenium) unless @driver
+          end
         end
 
         def driven_by(*args, &blk)
           @driver = ::ActionDispatch::SystemTestCase.driven_by(*args, &blk).tap(&:use)
```

Comment:
The `.use` method returns a :selenium symbol.
We use `.tap` to assign an instance of `SystemTesting::Driver` to `@driver` instead of a symbol.

https://github.com/rails/rails/blob/66cabeda2c46c582d19738e1318be8d59584cc5b/actionpack/lib/action_dispatch/system_testing/driver.rb#L16-L20